x-pack/plugins/cloud_security_posture/public/pages/compliance_dashboard/dashboard_sections/summary_section.tsx

```diff
@@ -124,7 +126,13 @@ export const SummarySection = ({
   );
 
   return (
-    <EuiFlexGroup gutterSize="l">
+    <EuiFlexGroup
+      gutterSize="l"
+      css={css`
+        // height for compliance by cis section with max rows
+        height: 310px;
+      `}
+    >
```

Comment:
unrelated but this fixes problems of unmatching summary sizes between dashboard tabs when there are not enough items in the compliance by score table